packages/api/es/indexer.js

```diff
@@ -156,21 +220,21 @@ function pdr(esClient, payload, index = 'cumulus', type = 'pdr') {
   stats.total = stats.processing + stats.completed + stats.failed;
   let progress = 0;
   if (stats.processing > 0 && stats.total > 0) {
-    progress = stats.processing / stats.total;
+    progress = ((stats.total - stats.processing) / stats.total) * 100;
```

Comment:
If I'm reading this right, this was going to count from 0% up to 100% as it was previously written.  It looks like it's now counting from 100% down to 0%?

Reply:
yeah, this was bug that I discovered. With the new formula the progress percentage increases as the number of processing granules go down.